src/lnd-actions/gen-seed.js

```diff
@@ -0,0 +1,20 @@
+const { deadline } = require('../grpc-utils')
+
+/**
+ * Generates a mnuemonic seed used to recover a user's wallet
+ *
+ * @function
+ * @see {@link http://api.lightning.community/#genSeed}
+ * @return {Object} res
+ * @return {Object} cipherSeedMnemonic
```

Comment:
dual return value is weird here, as is having the return value be anything other than a Promise

Reply:
This is a jsdoc mistake.